spec.html

```diff
@@ -11538,6 +11469,7 @@
         <emu-alg>
           1. Let _head_ be the TV of |TemplateHead| as defined in <emu-xref href="#sec-template-literal-lexical-components"></emu-xref>.
           1. Let _sub_ be the result of evaluating |Expression|.
+          1. ReturnIfAbrupt(_sub_).
```

Comment:
would it be useful (especially with property initializers) to make an abstract operation for "evaluate" that could be prefixed with `?`?